front/src/components/Auth/Login.vue

```diff
@@ -23,23 +25,22 @@
           label="Token"
         />
       </q-card-section>
-      <q-card-actions class="q-pa-md row justify-between">
-        <div v-if="registrationEnabled">
-          I don't have an account:
-          <b>
-            <ctf-note-link
-              name="auth-register"
-              class="text-primary"
-              label="REGISTER"
-              underline
-            />
-          </b>
-        </div>
-        <div v-else />
-        <q-btn type="submit" label="Login" color="primary" />
+
+      <q-card-actions class="row q-px-md q-pb-md">
+        <q-btn type="submit" label="Login" color="primary" class="full-width" />
       </q-card-actions>
     </q-form>
   </q-card>
+
+  <q-card class="q-mt-md">
+    <q-card-actions v-if="registrationEnabled" class="row q-px-md">
+      <span>Don't have an account?</span>
+      <q-space />
+      <ctf-note-link name="auth-register">
+        <q-btn flat color="primary">Register</q-btn>
+      </ctf-note-link>
+    </q-card-actions>
+  </q-card>
```

Comment:
The register button has a little too less contrast in dark mode in my opinion. Do you agree? Maybe an outlined button would catch more attention?

Reply:
Since this is the secondary option (most of the time you want to use the login button) I think it is fine.